platformio.ini

```diff
@@ -350,6 +349,26 @@ lib_ignore    = c1921b4
   TMC26XStepper
   U8glib-HAL
 
+#
+# MKS Robin nano (STM32F103VET6)
+#
+[env:mks_robin_mini]
+platform      = ststm32
+framework     = arduino
+board         = genericSTM32F103VE
+extra_scripts = buildroot/share/PlatformIO/scripts/mks_robin_mini.py
+build_flags   = !python Marlin/src/HAL/HAL_STM32F1/STM32F1_flag_script.py
+  ${common.build_flags} -std=gnu++14
```

Comment:
need to set the build_unflags to have effect

Reply:
Unflags! Nice one, Ivan K.